Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -394,7 +419,7 @@ function QuestieOptions.tabs.general:Initialize()
             },
             minLevelFilter = {
                 type = "range",
-                order = 15,
+                order = 17,
```

Comment:
Same as before: The order seems off. Maybe it is right though and the options look just fine?